docs/tutorial/vqe_for_hydrogen.ipynb

```diff
@@ -0,0 +1,397 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "plt.plot(cost_history)\n",
+    "plt.plot(\n",
```

Comment:
For this purpose, it is better to use `plt.hlines`
Moreover, it is better to add `plt.legend()` and axis labels